--- /mnt/data/Windsor_Widget_Row_Independent_YU_v1.5.0/yu_order_review_export_test_window.py	2026-07-16 23:32:59.418872379 +0000
+++ /mnt/data/Windsor_Widget_MYOB_No_Currency_Fields_v1.6.7/yu_order_review_export_test_window.py	2026-07-16 23:45:42.859834073 +0000
@@ -94,8 +94,6 @@
     "Freight Tax Code",
     "Freight Tax Amount",
     "Purchase Status",
-    "Currency Code",
-    "Exchange Rate",
     "Terms - Payment is Due",
     "           - Discount Days",
     "           - Balance Due Days",
@@ -231,8 +229,6 @@
             "Freight Tax Code": MYOB_YU_TAX_CODE,
             "Freight Tax Amount": "0.00",
             "Purchase Status": MYOB_YU_PURCHASE_STATUS,
-            "Currency Code": "",
-            "Exchange Rate": "",
             "Terms - Payment is Due": "",
             "           - Discount Days": "",
             "           - Balance Due Days": "",
